storybook/src/data/documents.data.js

```diff
@@ -0,0 +1,37 @@
+import uuid from 'uuid/v4';
+import image from './nuxeo-logo.jpg';
+
+export const documentGenerator = (numberOfDocs = 0) => {
+  const documents = [];
+  for (let i = 0; i < numberOfDocs; i++) {
+    const document = {
+      'entity-type': 'document',
+      uid: uuid(),
+      facets: [],
+      contextParameters: {
+        permissions: ['Write', 'ManageWorkflows'],
+      },
+    };
+    documents.push(document);
+  }
+  return documents;
+};
+
+export const DOCUMENT_DOWNLOAD = {
```

Comment:
From what we have today, it seems to me that we can put this in our generator, no?
The same for trashed documents, but in this case, under a flag.